Example/Tests/Tests.m

```diff
@@ -23,14 +25,51 @@ - (void)testSpecVersion {
     XCTAssertEqualObjects(self.spec.version, @"1.0.0");
 }
 
-- (void)testSpecSummary {
-    XCTAssertEqualObjects(self.spec.summary, @"A library with some functionality");
+- (void)testSpecAuthors {
+    XCTAssertEqualObjects(self.spec.authors, @[[[TMAuthor alloc] initWithName:@"Bryan Irace" emailAddress:@"bryan@irace.me"]]);
```

Comment:
Probably worth adding a couple of cases for the other ways of specifying authors.